### PR TITLE
Commit Config-File Test Data Info.

### PR DESCRIPTION
**Pull request by Vince505**
Added functionality on the datamining pipeline in order to get the location and parameters of the test data from the json config file.

Tests were also added to check the correct reading of the parameters from the json file.

@krsgpp

### CODE REVIEW

File: datadriven/src/sgpp/datadriven/datamining/modules/dataSource/DataSourceConfig.hpp [hunk 1]
```diff
@@ -35,70 +35,127 @@ enum class DataSourceShufflingType { random, sequential };
...
-   */
-  std::vector<double> readinClasses = std::vector<double>();
-  /**
-   * Specifies the set of columns (dimensions) to be read-in from the data file
-   * Starts at 0, order matters; Any column not contained in this vector is ignored
-   * as a dimension
-   * If empty, then all columns are read in (default)
-   */
-  std::vector<size_t> readinColumns = std::vector<size_t>();
+	/**
```
[the remark this answers is not in the document]
Comment (Vince505) [reply datadriven/src/sgpp/datadriven/datamining/modules/dataSource/DataSourceConfig.hpp]: Indentation corrected. The updated file is already pushed to the repository

**Comment by krsgpp** [comment]
[First jenkins run](https://simsgs.informatik.uni-stuttgart.de/jenkins/job/SGpp-meta-testing/32/)

**Comment by krsgpp** [comment]
[Second jenkins run](https://simsgs.informatik.uni-stuttgart.de/jenkins/job/SGpp-meta-testing/35/)

**Comment by krsgpp** [comment]
[Third jenkins run](https://simsgs.informatik.uni-stuttgart.de/jenkins/job/SGpp-meta-testing/44/)

**Comment by krsgpp** [comment]
[Fourth jenkins run](https://simsgs.informatik.uni-stuttgart.de/jenkins/job/SGpp-meta-testing/45/)

**Pull request merged**